### PR TITLE
Spruce up postgresql docs.

### LINKED ISSUE #736: Developer guide: Postgres plugin

### PR DESCRIPTION
**Pull request by stevenschlansker**
Fixes #736

### CODE REVIEW

**Comment by qualidafial** [comment]
Oh noes! I've totally restructured the asciidoc files on the `developer-guide` branch. That's ok, this shouldn't be too hard to integrate.

**Comment by stevenschlansker** [comment]
I'm happy to integrate it after yours lands.  Shouldn't be hard.

File: docs/src/adoc/postgresql.adoc [hunk 1]
```diff
@@ -1 +1,15 @@
-TODO
+*jdbi3-postgres* provides enhanced integration with the 
+link:https://jdbc.postgresql.org/[PostgreSQL JDBC Driver].
+
+[source,java,indent=0]
```
Comment (qualidafial) [inline docs/src/adoc/postgresql.adoc]: Heads up: the `indent=0` is only necessary when including a tagged from another file. Inline code doesn't need it.

File: postgres/src/main/java/org/jdbi/v3/postgres/JavaTimeArgumentFactory.java [hunk 2]
```diff
@@ -21,11 +21,15 @@
...
 import java.util.Optional;
 
-import org.jdbi.v3.core.config.ConfigRegistry;
 import org.jdbi.v3.core.argument.Argument;
 import org.jdbi.v3.core.argument.ArgumentFactory;
 import org.jdbi.v3.core.argument.ObjectArgument;
+import org.jdbi.v3.core.config.ConfigRegistry;
 
+/**
+ * Maps {@link LocalDate}, {@link LocalTime}, {@link LocalDateTime}, {@link OffsetDateTime}.
```
Comment (qualidafial) [inline postgres/src/main/java/org/jdbi/v3/postgres/JavaTimeArgumentFactory.java]: Nit: should probably be "Binds" not "Maps"

**Comment by qualidafial** [comment]
My changes are huge and I'm not nearly done. Go ahead and merge when you're ready; it'll be easier to integrate your changes into my branch, than the other way around.

**Comment by qualidafial** [comment]
I've integrated your changes into my developer guide branch: #802

**Pull request merged**